### ISSUE TITLE
Make page for contributions

### ISSUE THREAD

**Issue by roarfred** [roarfred/AmsToMqttBridge#25]

**Comment by Geving** [comment]
I just wanted to drop a tip for those who have issues with the WiFi signal strength inside the breaker cabinet; Get yourself an ESP-07 instead of the ESP-12. They both have the same ESP8266 chip, and the same form factor, but the ESP-07 has an U.FL Connector for an external antenna (as well as a proper ceramic antenna instead of only a PCB track like the ESP-12 has).

An external antenna using a U.FL connector would only need a 3-4mm hole through the cabinet wall, or you might just close the door over the thin antenna cable.

@roarfred Please feel free to leave, remove, replace or rewrite this text as you see fit.

**Comment by DIYglenn** [comment]
Would this work? The ESP-07 does not have the 6 programming pins at the bottom. The ESP-12E does have these 6, and it is soldered in the example...
With my fusebox outside the apartment, behind 30cm+ of concrete I might need some better signal. 

Update:
Nevermind, the schematics documents that these are not used. Probably just added for better support?

**Comment by stenjo** [comment]
Yes, this will work. The bottom pins are not in use on the pcb. I'm using ESP-12S that also does not have the bottom pins. Works perfectly :-)

**Comment by DIYglenn** [comment]
Great! The ESP-07 seems to be the best board for this application because of the better antenna, and the possibility of an external antenna.

![esp8266-esp-07-wifi-board-with-full-i-o-chip-antenna-and-full-external-antenna-bundle-from-tronixlabs-australia-032](https://user-images.githubusercontent.com/10504432/42551842-a5297e74-84d9-11e8-9ecf-4ce952e13d5b.jpg)

A note to anyone who'd like to use this board with an external antenna, remove the zero ohm SMD on it:
![esp07](https://user-images.githubusercontent.com/10504432/42551934-1ced967a-84da-11e8-9344-674f46c6181d.png)
